Auth0/CredentialsManager.swift

```diff
@@ -172,6 +172,16 @@ public struct CredentialsManager {
     }
     #endif
 
+    /// Retrieve credentials from keychain without any validation of the expiration.
+    /// No attempts to renew the credentials will be made so it's up to to caller to make the expiration date checks if they are relevant.
+    public var credentials: Credentials? {
```

Comment:
Sorry, I know I'm not a contributor to this repo. But as a user of this library and a library author myself, I'd recommend a better name here that's more indicative of its unsafe nature. Unsafe might be the wrong term, but hopefully you get my meaning 😬